ui/pages/espace-pro/creation/fin.tsx

```diff
@@ -137,7 +137,7 @@ function FinComponent(props) {
    */
   const onClose = async () => {
     await client.invalidateQueries("offre-liste")
-    await router.push(`/espace-pro/administration/entreprise/${encodeURIComponent(establishment_id.toString())}`)
+    await router.push(`/espace-pro/administration/entreprise/${establishment_id.toString()}`)
```

Comment:
```suggestion
    await router.push(`/espace-pro/administration/entreprise/${encodeURIComponent(establishment_id.toString())}`)
```
Ici, ça vaut le coup de le garder parce qu'on utilise pas notre client API